bytes/concat.ts

```diff
@@ -2,6 +2,7 @@
 // This module is browser compatible.
 
 import type { Uint8Array_ } from "./_types.ts";
+// deno-lint-ignore deno-std-docs/exported-symbol-documented
```

Comment:
Alternatively, we could have this rule ignore re-exported types. However, that would assume that re-exported types have documentation. In this codebase that might be true but would be a far reaching assumption in other codebases.

Reply:
Yes, by using [`@deno/doc`](https://jsr.io/@deno/doc). Though, I haven't tried it yet, and I assume it might come with a bit of a performance cost. IMHO, how it is now is fine enough to get the ball rolling on this lint plugin, and I think we could revisit this later. WDYT?

Reply:
Maybe it's fine to land if this is only informative, not requirement in CI.

> we could have this rule ignore re-exported types

Could you implement this for now? I think this directive would be confusing to contributors